File: C# Basics/Conditions/SpeedInfo.cs
using System;

namespace SpeedInfo
{
    class Program
    {
        static void Main(string[] args)
        {
            double speed = double.Parse(Console.ReadLine());

            string speedLabel = String.Empty;

            if (speed <= 10)
            {
                speedLabel = "slow";
            }
            else if (speed <= 50)
            {
                speedLabel = "average";
            }
            else if (speed <= 150)
            {
                speedLabel = "fast";
            }
            else if (speed <= 1000)
            {
                speedLabel = "ultra fast";
            }
            else
            {
                speedLabel = "extremely fast";
            }

            Console.WriteLine(speedLabel);

        }
    }
}

File: Java Basics/ConditionsEx/SpeedInfo.java
package ConditionsEx;

import java.util.Scanner;

public class SpeedInfo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double input = Double.parseDouble(scanner.nextLine());

        if (input <= 10) {
            System.out.println("slow");
        } else if (input > 10 && input <= 50) {
            System.out.println("average");
        } else if (input > 50 && input <= 150) {
            System.out.println("fast");
        } else if (input > 150 && input <= 1000) {
            System.out.println("ultra fast");
        } else {
            System.out.println("extremely fast");
        }
    }
}
